templates/zerver/api/get-stream-topics.md

```diff
@@ -0,0 +1,70 @@
+# Get topics in a stream
+
+Get all the topics in a specific stream
+
+`GET {{ api_url }}/v1/users/me/<stream_id>/topics`
+
+## Usage examples
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+<li data-language="javascript">JavaScript</li>
+<li data-language="curl">curl</li>
+</ul>
+<div class="blocks">
+
+<div data-language="curl" markdown="1">
+
+```
+curl {{ api_url }}/v1/users/me/<stream_id>/topics \
+    -u BOT_EMAIL_ADDRESS:BOT_API_KEY
+```
+
+</div>
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|get-stream-topics|example}
+
+</div>
+
+<div data-language="javascript" markdown="1">
+More examples and documentation can be found [here](https://github.com/zulip/zulip-js).
+```js
+const zulip = require('zulip-js');
+
+// Download zuliprc-dev from your dev server, assuming it's in the local dir
+const config = {
+    zuliprc: 'zuliprc-dev',
+};
+
+zulip(config).then((client) => {
+    // Get all the topics in stream with ID 1
+    return client.streams.topics.retrieve({ stream_id: 1 });
+}).then(console.log);
+
+```
+</div>
+
+</div>
+
+</div>
+
+## Response
+
+#### Return values
+
+* `topics`: An array of `topic` objects, which contain:
+    * `name`: The name of the topic.
+    * `max_id`: The message ID of the last message sent to this topic.
+
+#### Example response
+
+A typical successful JSON response may look like:
+
+{generate_code_example|get-stream-topics|fixture}
+
+An example of a JSON response for when the user is not authorized
```

Comment:
`An example JSON response...` is better/more concise than `An example of a JSON response...`